compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -132,22 +122,45 @@ static LogicalResult setRootConfigForPackPipeline(func::FuncOp entryPointFn,
 
   SmallVector<PackingConfigPackingLevelAttr> packingConfigLevelsVal = {
       packingConfigLevel1Attr, packingConfigLevel2Attr};
-
   auto packingConfigLevels =
       PackingConfigPackingLevelsAttr::get(context, packingConfigLevelsVal);
   auto config = PackingConfigAttr::get(context, packingConfigLevels);
   setPackingConfig(linalgOp, config);
+
+  // ------------------------------------------------------
+  // -------------- Set lowering config -------------------
+  // ------------------------------------------------------
+  // Currently, assume working on a 2x2 AIE array, so the second level tile
+  // sizes should be (tileM0/2, tileN0/2). Considering the packing sizes, the
+  // adjusted tile sizes should be (tileM0/2/packedM1, tileN0/2/packedN1).
+  auto packedM1 = packedSizes[3];
```

Comment:
Can you also add a lit test for this?

https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lowering_strategy.mlir

Reply:
Done